### PR TITLE
Update cross entropy documentation to metion logits clearly

### PR DESCRIPTION
**Pull request by Alex7Li**
### Description
Improved the documentation for cross entropy as it is a common point of confusion.

### Issue
#82081 

### Testing
I did not test this change as it is tiny and documentation-only

### CODE REVIEW

File: torch/nn/modules/loss.py [hunk 1]
```diff
@@ -1029,14 +1029,17 @@ def forward(self, input: Tensor, target: Tensor) -> Tensor:
...
+    and target.
 
     It is useful when training a classification problem with `C` classes.
     If provided, the optional argument :attr:`weight` should be a 1D `Tensor`
     assigning weight to each of the classes.
     This is particularly useful when you have an unbalanced training set.
 
-    The `input` is expected to contain raw, unnormalized scores for each class.
+    The `input` is expected to contain the logits for each class (which do `not` need
+    to be positive or sum to 1, in general). This class will compute a softmax
```
Comment (jbschlosser) [inline torch/nn/modules/loss.py:1041]: note that a LogSoftmax is actually computed, not a softmax. I suggest leaving out this sentence

**Review by jbschlosser — COMMENTED** [review]
Thanks for the update!

File: torch/nn/modules/loss.py (hunk 1 above)
Comment (Alex7Li) [reply torch/nn/modules/loss.py]: Removed

File: torch/nn/modules/loss.py [hunk 2]
```diff
@@ -1029,14 +1029,17 @@ def forward(self, input: Tensor, target: Tensor) -> Tensor:
...
+    r"""This criterion computes the cross entropy loss between input logits
+    and target.
 
     It is useful when training a classification problem with `C` classes.
     If provided, the optional argument :attr:`weight` should be a 1D `Tensor`
     assigning weight to each of the classes.
     This is particularly useful when you have an unbalanced training set.
 
-    The `input` is expected to contain raw, unnormalized scores for each class.
+    The `input` is expected to contain the logits for each class (which do `not` need
```
[the remark this answers is not in the document]
Comment (Alex7Li) [reply torch/nn/modules/loss.py]: I used 'unnormalized logits' in both places, since I don't think 'scores' means anything

**Review by jbschlosser — APPROVED** [review]
LGTM thanks for the fix!
